h/dynalloc.h

```diff
@@ -696,11 +741,23 @@ typedef struct DynallocInputParms_tag {
   char reserved[3];
 } DynallocInputParms;
 
+
+typedef struct DynallocNewInputParms_tag {
+  int type;
```

Comment:
* What do I use in `type`?
* Are the value in `key` the same as the ones used for real text units?

It looks like a leaky abstraction:
* In order to use this function I have to study the source code
* I end up using text units anyways, because this is what this `DynallocNewInputParms` is, sort of.

Reply:
Resolved. Used TextUnit structure instead of DynallocNewInputParms.